### ISSUE TITLE
USING LR REDUCE LR ON PLATEAU

### ISSUE THREAD

**Issue by massimilianoporzio** [Project-MONAI/MONAI#1362]
In the file test_handler_lr_scheduler.py  :
  lr_scheduler = torch.optim.lr_scheduler.ReduceLROnPlateau(optimizer, patience=1)
  handler = LrScheduleHandler(lr_scheduler, step_transform=lambda x: val_engine.state.metrics["val_loss"])

In the code of challenge_baseline how to use it? Can anyone explain what to specify in step_transform and output_transform?
evaluator.state.metrics["val_loss"] does not work.

**Comment by Nic-Ma** [comment]
Hi @wyli ,

Could you please help share some info here? I think may need to change the expected metric key in `step_transform` according to `challenge_baseline` notebook.

Thanks.

**Comment by massimilianoporzio** [comment]
THE problem is that the trainer is a MONAI trainer not a Ignite trainer and
is training using a dictionary based dataset so I think I have to change
something but I don't know  exactly what...
As soon as possible I will share a code chunk with errors....

Il giorno lun 14 dic 2020 alle ore 09:57 Nic Ma <notifications@github.com>
ha scritto:

**Comment by Nic-Ma** [comment]
Sure, please share your code for better debug and analysis.
Thanks.
